presto-spi/src/main/java/com/facebook/presto/spi/security/AccessDeniedException.java

```diff
@@ -228,14 +228,14 @@ public static void denyRevokeTablePrivilege(String privilege, String tableName,
         throw new AccessDeniedException(format("Cannot revoke privilege %s on table %s%s", privilege, tableName, formatExtraInfo(extraInfo)));
     }
 
-    public static void denyShowGrants(String user)
+    public static void denyShowGrants()
     {
-        denyShowGrants(user, null);
+        denyShowGrants(null);
     }
 
-    public static void denyShowGrants(String user, String extraInfo)
+    public static void denyShowGrants(String extraInfo)
```

Comment:
It looks like you're only passing in the table name as "extraInfo" -- can you change the variable name to "tableName"? It still doesn't make sense to add "on" for any arbitrary string (which is what extraInfo suggests)